README.md

```diff
@@ -1,37 +1,39 @@
 > WIP
 
-# Radix Monorepo
+# Radix
 
 This repository is a [monorepo](https://github.com/babel/babel/blob/master/doc/design/monorepo.md) powered by [Lerna](https://github.com/lerna/lerna) and [yarn workspaces](https://yarnpkg.com/lang/en/docs/cli/workspaces/).
 
-### Develop
+## Contributing
 
-#### Install dependencies
+There are many ways to contribute to the Radix and it's packages. Please first [create an issue](https://github.com/modulz/radix/issues/new) with your proposal, and from there we can start a conversation.
 
-```sh
-yarn
-```
+#### Getting Started
 
-#### Run website + Radix
+Bootstrap the repo by simply running:
 
-```sh
-yarn start
+```shell
+yarn             # Installs dependencies and links packages
+yarn start       # Develop Radix & it's Website
 ```
 
-### Build & Release
+**Other Packages**
 
-#### Build
+Check the `readme.md` of other packages, like `@modulz/radix-icons`, to get more details on contributing.
 
-```sh
-yarn build
-```
+#### Publishing
 
-#### Release
+We use `lerna publish` for managing the versions of packages in the monorepo. Since we also use [conventional-commits](https://www.conventionalcommits.org), changelogs will be generated automatically.
 
-```sh
-yarn release
+Once all commits have been made, from the root of the monorepo simply run:
+
+```shell
+yarn release         # prompts modified packages, generates changelogs and publishes to npm
+yarn release:github  # does same as above, but also generates a github release
```

Comment:
Not sure about this? It seems all the releases I've been making are also on Github: https://github.com/modulz/radix/releases

Reply:
Ah ok it makes them "official" releases. 

Still wanna have some time to think about this. If we do decide to do it, I don't think it should be a manual thing. When we do `yarn release` it should do this automatically to ensure the releases on npm are consistent with the ones on GH.

Reply:
Rename `release:github` to `release`, then you're gtg on single command approach